aiohttp/web_urldispatcher.py

```diff
@@ -1239,6 +1297,55 @@ def freeze(self) -> None:
         super().freeze()
         for resource in self._resources:
             resource.freeze()
+        self._rebuild()
+
+    def _rebuild(self) -> None:
+        self._hyperdb = None
+        self._plain_resources.clear()
+        del self._prefix_resources[:]
+        patterns: list[bytes] = []
+        ids: list[int] = []
+        for id_, resource in enumerate(self._resources):
+            if isinstance(resource, PlainResource):
+                self._plain_resources[resource.get_info()["path"]] = resource
+                continue
+            elif isinstance(resource, DynamicResource):
+                pattern = resource.get_info()["pattern"].pattern
+            elif isinstance(resource, PrefixResource):
+                if isinstance(resource, MatchedSubAppResource):
+                    # wildcard resources doesn't fit hyperscan table
```

Comment:
This could probably work with a second hyperscan table. Though it's such a niche feature, it may not be worth the effort.

Reply:
I'd prefer keeping this possible optimization out of the PR scope if you don't mind.